lte/gateway/python/magma/subscriberdb/crypto/milenage.py

```diff
@@ -52,7 +52,7 @@ def generate_eutran_vector(self, key, opc, sqn, plmn):
 
         autn = Milenage.generate_autn(sqn_bytes, ak, mac_a, self.amf)
         kasme = Milenage.generate_kasme(ck, ik, plmn, sqn_bytes, ak)
-        return rand, xres, autn, kasme
+        return rand, xres, autn, kasme, ck, ik
```

Comment:
remove this modification and write a new function to generate 5G authentication vector in line with my other comments.

Reply:
comment resolution is part 1st PR, will include separately